tests/Log/SimpleLoggerTest.php

```diff
@@ -146,6 +150,9 @@ public function testPsrLogInterfaceMethods($message, string $expected): void
 
     public function testLogWithResourceMessage(): void
```

Comment:
```suggestion
    /**
     * @requires PHP >= 8.0
     */
    public function testLogWithResourceMessage(): void
```

Reply:
https://phpunit.readthedocs.io/en/9.5/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests-skipping-tests-examples-databaseclassskippingtest-php